Tue Jun 20 20:48:10 CEST 2006  VMiklos <vmiklos@frugalware.org>
  * libfwgrubconfig: fix for os-prober-38238
diff -rN -u old-frugalwareutils/libfwgrubconfig/libfwgrubconfig.c new-frugalwareutils/libfwgrubconfig/libfwgrubconfig.c
--- old-frugalwareutils/libfwgrubconfig/libfwgrubconfig.c	2006-06-20 21:02:20.000000000 +0200
+++ new-frugalwareutils/libfwgrubconfig/libfwgrubconfig.c	2006-06-20 21:02:20.000000000 +0200
@@ -508,7 +508,7 @@
 			if(stat("/usr/bin/linux-boot-prober", &buf))
 				continue;
 
-			ptr = g_strdup_printf("linux-boot-prober %s", entry->rootdev);
+			ptr = g_strdup_printf("linux-boot-prober --mounted %s", entry->rootdev);
 			pp = popen(ptr, "r");
 			free(ptr);
 			if(!pp)

